File: Python Code/Daily_Flash_Code_0/Week 5/11 Feb/25-DailyFlash_Solutions/25-DailyFlash_Solutions/11_Feb_Solution_Three/C++/prog2.cpp
#include<iostream>
#include<string.h>

int main() {
	
	char arr[10];
	printf("Enter octal : ");
	std::cin.getline(arr, sizeof(arr));
	int len = strlen(arr);
	int i = 0;
	int dec = 0;
	int num;
	while(1) {
		if(i > len - 1)
			break;
		num = arr[i] - 48;
		dec = dec * 8 + num;
		i++;
	}	
	std::cout << "Decimal : " << dec;
	return 0;
}	

File: Python Code/Daily_Flash_Code_0/Week 5/11 Feb/25-DailyFlash_Solutions/25-DailyFlash_Solutions/11_Feb_Solution_Three/Python/prog2.py
arr = input("Enter octal : ")
l = len(arr)
i = 0
dec = 0
while(True):
    if(i > l - 1):
        break
    num = ord(arr[i])- 48
    dec = dec * 8 + num
    i += 1
print("Decimal : ", dec)
